### PR TITLE
fix: update ubi base image

### PR DESCRIPTION
**Pull request by dimakis**
## Description

Updates base ubi image

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by dimakis** [comment]
@anishasthana @astefanutti @zdtsw 

as per this [conversation](https://redhat-internal.slack.com/archives/C05NXTEHLGY/p1697548522169089?thread_ts=1697036534.719189&cid=C05NXTEHLGY)

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19.10-10 as builder
+FROM registry.redhat.io/ubi8/go-toolset:1.19 as builder
```
Comment (astefanutti) [inline Dockerfile:2]: Don't we want to keep `registry.access.redhat.com`?

**Review by astefanutti — COMMENTED** [review]
Should this go upstream first?

File: Dockerfile (hunk 1 above)
Comment (dimakis) [reply Dockerfile:2]: I've asked if that was the right image in the thread? I'm unsure?

Comment (astefanutti) [reply Dockerfile:2]: My understanding is the thread might refer to downstream. Better double checking.

**Review by zdtsw — APPROVED** [review]
/lgtm

**Review by astefanutti — APPROVED** [review]
This patch can be removed once the fork is rebased.

**Pull request merged**